tools/nxstyle.c

```diff
@@ -386,24 +467,26 @@ int main(int argc, char **argv, char **envp)
               {
                 if (!bcrs)
                   {
-                    fprintf(stderr, "Carriage returns found.  "
-                            "First detected at line %d:%d\n", lineno, n);
+                    ERROR("Carriage returns found.  "
+                            "First detected", lineno, n);
                     bcrs = true;
                   }
               }
               break;
 
             default:
               {
-                fprintf(stderr,
-                        "Unexpected white space character %02x found at line %d:%d\n",
-                        line[n], lineno, n);
+                 snprintf(buffer, sizeof(buffer),
```

Comment:
why we can't ERROR directly?

Reply:
@xiaoxiang781216 - 
To make this conform to a very simple API and not lose any of Greg's work in formatting.

I will not be investing any more time on this.  
Please feel free to have your team make any further changes like adding var args it xiaimi needs them.